data/config-schema/src/main/resources/net/corda/schema/configuration/membership/1.0/corda.membership.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://corda.r3.com/net/corda/schema/configuration/membership/1.0/corda.membership.json",
+  "title": "Corda Membership Configuration Schema",
+  "description": "Configuration schema for the membership subsection.",
+  "type": "object",
+  "default": {},
+  "properties": {
+    "maxDurationBetweenSyncRequestsMinutes": {
+      "description": "Maximal duration in minutes between member list synchronization requests.",
+      "type": "integer",
+      "minimum": 10,
```

Comment:
10 seems like a realistic minimum, but do you think someone may want to set lower for testing?

Reply:
I've changed the code to test it.